### DISCUSSION TITLE
apps/upgrade-inplace

### DISCUSSION THREAD

**Discussion by giscus** [imroc/tke-guide#16] [General]
# apps/upgrade-inplace

需求与背景

https://imroc.cc/tke/apps/upgrade-inplace

**Comment by ermazi** [comment]
哈铪, tke中推 openkruise ~

**Reply by imroc** [reply]
哈哈, 非官方文档, 个人从实用角度写的

**Comment by ermazi** [comment]
确实想引入原地升级, 但是考虑到 超级节点 不支持, 果断放弃; 不知道啥时候能 超级

**Reply by ermazi** [reply]
节点也能适配
